src/content/blog/release-of-ada-v3.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: 'Release of Ada v3.0 with URLPattern'
+description: >-
+  Announcing Ada's new release with URLPattern, a new feature that allows you to
+  define URL patterns for your routes.
+type: Blog
+date: '2025-01-30'
+tag: performance
+status: published
+---
+
+[We are excited][contributors] to announce the release of Ada v3.0.
+This major release includes a lot of new features, improvements and bug fixes.
+
+Since our last release (v2.9.2), published on September 2, 2024,
+[there have been 262 commits written by 9 contributors][diff-between-292].
+Ada is now used by major products and softwares like Node.js, ClickHouse,
+Redpanda, Kong, Telegram and Cloudflare Workers.
+
+## New features
+
+- Ada now supports Bazel build system.
+- Ada now supports Unicode 15.1.0 through [ada_idna][ada-idna] library.
+- Ada now includes an experimental URLPattern implementation that allows
+you to define URL patterns for your routes, and match them.
+  - It doesn't provide a regex engine and leaves the decision
+  of choosing the right engine to the user. This is done as a security measure
+  since the default std::regex engine is not safe and open to DDOS attacks.
```

Comment:
I suggest to explain what this actually means. Something like: 

If your UrlPatterns come from untrusted source (like a user), then DO NOT use std::regex 
in production. Unsafe regular expression languages have unbounded memory consumption
and execution times. The only safe implementations known to authors at this time are "v8" and "re2"

Reply:
Updated. Thanks!